backend/src/routers/index.ts

```diff
@@ -3,4 +3,5 @@ export { exportRouter } from './export';
 export { healthRouter } from './health';
 export { userRouter } from './user';
 export { presetsRouter } from './presets';
+export { getAdminRouter } from './admin';
```

Comment:
Minor thing, but just "adminRouter" would be more consistent with the other exports

Reply:
Unlike other routers it's behind an async function you have to await so I think "get" makes sense for it.